folders/backend/express-routes/server.ts

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+
+const app = express();
+
+app.use((req, res, next) => {
+  console.log('Hello, World!');
+  console.log('The date is', new Date());
+  next();
+});
+
+app.get('/', (req, res) => {
+  res.send('Here are your notes.');
+});
+
+app.get('/notes', (req, res) => {
+  res.send('Here are your notes.');
```

Comment:
Since / and /notes send the same response, how do you know you got the right one?